pkg/cmd/admin/network/project_options.go

```diff
@@ -84,7 +84,14 @@ func (p *ProjectOptions) Validate() error {
 		errList = append(errList, errors.New("must provide --selector=<project_selector> or projects"))
 	}
 
-	// TODO: Validate if the openshift master is running with mutitenant network plugin
```

Comment:
If the user is not using openshift-sdn plugin then "default" cluster network resource won't be present and this validation will be incorrect. So if the error returned from Get(sdnapi.ClusterNetworkDefault) is kerrs.IsNotFound or if the plugin is not openshift multitenant, we should throw an error with message 'managing pod network is only supported for openshift multitenant network plugin'.

For the second issue, AFAIK we don't expect old api server to return ClusterNetwork resource without pluginName as we only allow incremental upgrades i.e. 3.2 to 3.3 and then 3.3 to 3.4 release (https://github.com/openshift/origin/pull/11073#issuecomment-250283774). PluginName field was added in 3.3 release, so this validation in 3.4 can expect valid plugin name if present.